modules/virt-vm-behavior-dr.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-disaster-recovery.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="virt-vm-behavior-dr_{context}"]
+= VM behavior during disaster recovery scenarios
+
+VMs typically act similarly to pod-based workloads during failover and relocate disaster recovery flows.
+
+[discrete]
+[id="dr-failover_{context}"]
+== Failover
+
+Use failover when there is a critical failure in the primary environment that makes it impractical or impossible to use relocation to move the workload to a secondary environment. When failover is executed, the storage is fenced from the primary environment, the I/O to the VM disks is abruptly halted, and the VM restarts in the secondary environment using the replicated data.
```

Comment:
**suggestion**: Use failover when there is a critical failure in the primary environment, making it impractical or impossible to relocate the workload to a secondary environment.

Reply:
**thought**: Can we rewrite the "When failover is exe..." sentence in active voice? Who does these action?